interceptor.go

```diff
@@ -235,17 +235,45 @@ func (i *interceptor) WrapStreamingClient(next connect.StreamingClientFunc) conn
 			protocol: protocolToSemConv(conn.Peer()),
 			attrs:    attributesFromRequest(req),
 		}
+		carrier := propagation.HeaderCarrier(conn.RequestHeader())
+		ctx = i.config.propagator.Extract(ctx, carrier)
```

Comment:
.propagator.Extract should only be used in initial call in the server interceptor...